common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -9,6 +9,79 @@
 @elite_ranks = "gfx/interface/icons/culture_innovations/innovation_elite_ranks.dds"
```

Comment:
We had a conversion about it. You decided to keep giants?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108394902-ac513f80-722e-11eb-9742-b0ea9b56aa23.png)

</details>

Reply:
For those cultures that won't have a specific cultural unit.